app/src/main/java/org/sil/storyproducer/controller/MainActivity.kt

```diff
@@ -101,10 +103,31 @@ class MainActivity : BaseActivity(), Serializable {
             }
         }
 
+        checkDownloadStoriesMessage()
```

Comment:
Peter, please help me understand why this snackbar message is best implemented here in the MainActivity and relying on the Workspace.storyFilesToScanOrUnzip() which looks at the files to be installed, rather than implemented in StoryPageFragment.kt when it sets up the view of the tab (and can check Workspace.Stories size). I had hoped to ask you this yesterday, but we didn't have time then. We could talk Monday? Thanks!

Reply:
@ithinkapps Hi Peter, please update the message to add the word Tip, and push the branch to your fork. That should automatically trigger an update to PR 669. Thanks!